api/Dockerfile

```diff
@@ -1,5 +1,5 @@
 # This must be run in project root, to pick up dataset_config/.
-FROM python:2
+FROM python:3
```

Comment:
Sorry unrelated: in `dataset_config/framingham_heart_study_teaching/ui.json` `search_placecholder_text`, do you mind making "bmi" and "stroke" lowercase? Thanks.